### ISSUE TITLE
Button examples break WCAG 2.5.2 'Pointer Cancellation'

### ISSUE THREAD

**Issue by jfhector** [w3c/aria-practices#1217]
#### Summary of the issue

The two buttons in the [Button Examples page](http://w3c.github.io/aria-practices/examples/button/button.html) get activated on `keydown` events when the key that is pressed is 'Enter'.

This breaks [WCAG Success Criterion 2.5.2 'Pointer Cancellation'](https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation.html).

##### Issue in the code for action button

In the [source JavaScript code for the button examples](http://w3c.github.io/aria-practices/examples/button/js/button.js), the `activateActionButton` function is called inside `actionButtonKeydownHandler` if the key that was pressed is 'Enter'.

`activateActionButton` should be called in `actionButtonKeyupHandler` instead.

##### Issue in the code for toggle button

Similarly, the `toggleButtonState` function is called inside `toggleButtonKeydownHandler` if the key that was pressed is 'Enter'.

`toggleButtonState` should be called in `toggleButtonKeyupHandler` instead.

#### Fixing the issue

I couldn't find another report of this issue. If you confirm that there isn't already a PR for this, I'm very happy to submit one.

Thanks for all your great work on the ARIA Authoring Practices. They've been an invaluable resource for me.

**Comment by ZoeBijl** [comment]
Thank you for your detailed issue report.

The success criterion reads:

> For functionality that can be operated using a single pointer, at least one of the following is true:

Where a single pointer is defined as:

> pointer input that operates with one point of contact with the screen, including single taps and clicks, double-taps and clicks, long presses, and path-based gestures

Do you think a keyboard/the enter key fits that definition?

**Comment by jfhector** [comment]
You are right @ZoeBijl, 2.5.2 doesn't apply to keyboard events.

Apologies. I submitted this too quickly.

**Comment by ZoeBijl** [comment]
No need to apologise. Better to have an easy to solve/close bug than a hard one that doesn’t get reported at all 🌟.
